<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>区块链电子病历</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/src/Resources/icon.png</normaloff>:/new/src/Resources/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTreeWidget" name="treeWidget">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <column>
            <property name="text">
             <string notr="true">病人列表</string>
            </property>
           </column>
          </widget>
          <widget class="Line" name="line_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>5</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QListWidget" name="listWidget">
           <item>
            <property name="text">
             <string>病人信息</string>
            </property>
           </item>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <property name="documentMode">
           <bool>true</bool>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>false</bool>
          </property>
          <property name="tabBarAutoHide">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>历史病历</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QTextEdit" name="textEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>参考模板</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,0">
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>100</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QTextEdit" name="textEdit_2">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>720</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>100</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>其他文书</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <spacer name="verticalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>100</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QTextEdit" name="textEdit_3">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>720</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>操作员：主治医师</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>400</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="timelable">
          <property name="text">
           <string>当前时间</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resource.qrc">:/new/src/Resources/HFUT.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>65535</width>
            <height>65535</height>
           </size>
          </property>
          <property name="text">
           <string>合肥工业大学计算机与信息学院信安大创项目组</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBa">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1013</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>病历</string>
    </property>
    <addaction name="actionAddPatient"/>
    <addaction name="actionNew"/>
    <addaction name="actionCase"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionUpload"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>表格</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>系统</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>模板</string>
    </property>
    <addaction name="actionTemplateNew"/>
    <addaction name="actionTemplateOpen"/>
    <addaction name="actionTemplateSave"/>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_6"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_7"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionUpload"/>
   <addaction name="separator"/>
   <addaction name="actionDel"/>
   <addaction name="separator"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionTemplate"/>
   <addaction name="separator"/>
   <addaction name="actionShowCard"/>
   <addaction name="separator"/>
   <addaction name="actionTrans"/>
   <addaction name="separator"/>
   <addaction name="actionTempe"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionnew_record">
   <property name="text">
    <string>new record</string>
   </property>
  </action>
  <action name="actiondelete_record">
   <property name="text">
    <string>delete record</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/src/Resources/new.png</normaloff>:/new/src/Resources/new.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/src/Resources/save.png</normaloff>:/new/src/Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/src/Resources/upload.png</normaloff>:/new/src/Resources/upload.png</iconset>
   </property>
   <property name="text">
    <string>上传</string>
   </property>
  </action>
  <action name="actionNewTree">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/src/Resources/tree.png</normaloff>:/new/src/Resources/tree.png</iconset>
   </property>
   <property name="text">
    <string>新建树测试</string>
   </property>
   <property name="toolTip">
    <string>新建树</string>
   </property>
  </action>
  <action name="actionNewInfo">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/src/Resources/new.png</normaloff>:/new/src/Resources/new.png</iconset>
   </property>
   <property name="text">
    <string>病人信息测试</string>
   </property>
   <property name="toolTip">
    <string>病人信息</string>
   </property>
  </action>
  <action name="actionShowCard">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/src/Resources/card.png</normaloff>:/new/src/Resources/card.png</iconset>
   </property>
   <property name="text">
    <string>显示卡片</string>
   </property>
   <property name="toolTip">
    <string>卡片</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/src/Resources/print.png</normaloff>:/new/src/Resources/print.png</iconset>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
   <property name="toolTip">
    <string>打印</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/src/Resources/delete.png</normaloff>:/new/src/Resources/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>delete</string>
   </property>
  </action>
  <action name="actionTrans">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/src/Resources/car.png</normaloff>:/new/src/Resources/car.png</iconset>
   </property>
   <property name="text">
    <string>转科</string>
   </property>
  </action>
  <action name="actionTempe">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/src/Resources/temperature.png</normaloff>:/new/src/Resources/temperature.png</iconset>
   </property>
   <property name="text">
    <string>体温</string>
   </property>
  </action>
  <action name="actionTemplateOpen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/src/Resources/templatesUse.png</normaloff>:/new/src/Resources/templatesUse.png</iconset>
   </property>
   <property name="text">
    <string>打开模板</string>
   </property>
  </action>
  <action name="actionTemplateNew">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/src/Resources/templateAdd.png</normaloff>:/new/src/Resources/templateAdd.png</iconset>
   </property>
   <property name="text">
    <string>新建模板</string>
   </property>
  </action>
  <action name="actionTemplate">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/src/Resources/template.png</normaloff>:/new/src/Resources/template.png</iconset>
   </property>
   <property name="text">
    <string>模板</string>
   </property>
  </action>
  <action name="actionTemplateSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/src/Resources/templateSave.png</normaloff>:/new/src/Resources/templateSave.png</iconset>
   </property>
   <property name="text">
    <string>保存模板</string>
   </property>
  </action>
  <action name="actionAddPatient">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/new/src/Resources/addPatient.png</normaloff>:/new/src/Resources/addPatient.png</iconset>
   </property>
   <property name="text">
    <string>病人登记</string>
   </property>
  </action>
  <action name="actionCase">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Resources/refresh.png</normaloff>Resources/refresh.png</iconset>
   </property>
   <property name="text">
    <string>刷新病历</string>
   </property>
   <property name="toolTip">
    <string>刷新病历</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
